app/src/debug/kotlin/io/homeassistant/companion/android/developer/catalog/HAComposeCatalogScreen.kt

```diff
@@ -82,23 +70,30 @@ fun HAComposeCatalogScreen() {
             topBar = {
                 TopBar { currentVariant = it }
             },
-        ) {
+            bottomBar = {
+                NavigationBar {
```

Comment:
The color of the nav bar in dark mode is not very nice. We don't plan to implement a HANavBar but you can still use better colors from `LocalHAColorScheme` for it.